app/views/protocols/form/_project_roles_fields.html.haml

```diff
@@ -29,7 +29,7 @@
       .form-group.row
         = pr_form.label :identity_id, t(:protocols)[:authorized_users][:primary_pi], class: 'col-lg-2 control-label required'
         .col-lg-10
-          = pr_form.text_field :identity_id, placeholder: t(:constants)[:prompts][:search_for_user], class: 'form-control'
+          = pr_form.text_field :identity_id, placeholder: t(:constants)[:prompts][:search_for_user], class: 'form-control rm-id-dependent'
```

Comment:
Needs to be added to the same partial in the dashboard
app/views/dashboard/protocols/form/_project_roles_fields.html.haml